File: anagram.py
#The main task here is to create a function to check if two strings are anagrams

def check_anagram(first_string, second_string):
#since strings in python are case sensitive we convert the first string and second to uppercase or lower
#we rearrange the characters in the string them next

     ordered_first_string = sorted(first_string.lower())
     ordered_second_string = sorted(second_string.lower())

     return ordered_first_string == ordered_second_string
#the function returns a bool True/ False

#for simplicity, we can create another function that will print the truth value
def print_anagram_status(string_1, string_2):
     print(check_anagram(string_1,string_2))
     return 0
     

print_anagram_status("listen","silent")
    

File: anagram.js
// Function to check if two strings are anagrams
function areAnagrams(str1, str2) {
  // Remove any non-alphabetic characters and convert to lowercase
  const cleanStr1 = str1.replace(/[^a-zA-Z]/g, '').toLowerCase();
  const cleanStr2 = str2.replace(/[^a-zA-Z]/g, '').toLowerCase();

  // Sort the characters in both strings
  const sortedStr1 = cleanStr1.split('').sort().join('');
  const sortedStr2 = cleanStr2.split('').sort().join('');

  // Compare the sorted words
  return sortedStr1 === sortedStr2;
}

// Accept two strings from the user
const string1 = prompt('Enter the first word:');
const string2 = prompt('Enter the second word:');

// Check if the strings are anagrams
if (areAnagrams(string1, string2)) {
  console.log('These words are anagrams.');
} else {
  console.log('These words are not anagrams.');
}